File: point.py
import math


class Point():

	def __init__(self, x, y):
		self.x = x
		self.y = y


	def distance_to(self, p):
		x = p.x - self.x
		y = p.y - self.y
		x *= x
		y *= y

		return math.sqrt(x + y)

File: scripts/point.js
class Point {

  constructor(x, y, id) {
    this.id = id;
    this.x = x;
    this.y = y;
    this.points = []
    this.distance = []
  }

  distance_to(p) {
    let x = p.x - this.x;
		let y = p.y - this.y;
		x *= x;
    y *= y;
    return Math.sqrt(x + y);
  }
}